igvc_rviz_plugins/plugin_description.xml

```diff
@@ -15,4 +15,12 @@
         </description>
     </class>
 
+    <class name="igvc_rviz_plugins/WaypointPanel"
+           type="igvc_rviz_plugins::WaypointPanel"
+           base_class_type="rviz::Panel"
```

Comment:
You are missing a ">" to finish the opening tag of the _class_ element here. This is causing the XML file to be malformed, causing rviz to be unable to read it.